ports/vcpkg-python-test-import/vcpkg-port-config.cmake

```
@@ -0,0 +1,65 @@
+include_guard(GLOBAL)
+include("${CURRENT_HOST_INSTALLED_DIR}/share/vcpkg-get-python/vcpkg-port-config.cmake")
+
+function(vcpkg_python_test_import)
+  cmake_parse_arguments(
+    PARSE_ARGV 0
+    "arg"
+    ""
+    "MODULE;PYTHON_EXECUTABLE"
+    "MODULES;DLL_SEARCH_PATHS"
+  )
+
+  if(VCPKG_CROSSCOMPILING OR (VCPKG_CRT_LINKAGE STREQUAL "static" AND NOT arg_PYTHON_EXECUTABLE))
+    return()
+  endif()
+
+  if(NOT arg_PYTHON_EXECUTABLE AND COMMAND vcpkg_get_vcpkg_installed_python)
+    vcpkg_get_vcpkg_installed_python(arg_PYTHON_EXECUTABLE)
+  elseif(NOT DEFINED arg_PYTHON_EXECUTABLE)
+    message(FATAL_ERROR "arg_PYTHON_EXECUTABLE is not defined and command vcpkg_get_vcpkg_installed_python is not available!")
+  endif()
+
+  if(arg_MODULE)
+    list(APPEND arg_MODULES "${arg_MODULE}")
+  endif()
+
+  set(PYTHON_DLL_SEARCH_DIRS "")
+  if(VCPKG_TARGET_IS_WINDOWS)
+    list(APPEND arg_DLL_SEARCH_PATHS "${CURRENT_INSTALLED_DIR}/bin")

```

```suggestion
    list(APPEND arg_DLL_SEARCH_PATHS "${CURRENT_INSTALLED_DIR}/bin")
    list(APPEND arg_DLL_SEARCH_PATHS "${CURRENT_PACKAGES_DIR}/bin")
```

dlls built by the current app also may need loading (e.g. gdal)